ts/features/mvl/saga/index.ts

```diff
@@ -25,3 +26,14 @@ export function* watchMvlSaga(bearerToken: SessionToken): SagaIterator {
     }
   );
 }
+
+// after making the features.MVL non-persistent
+// because it is no longer necessary, the persist:mvl
+// must be manually removed from AsyncStorage
+export function* removePersistMvl() {
+  try {
+    yield* call(AsyncStorage.removeItem, "persist:mvl");
+  } finally {
+    // nothing to do
```

Comment:
Maybe it is better to explicitly explain why we are not interested in handling any error?